src/common/transformations/tests/op_conversions/convert_maxpool_downgrade_test.cpp

```diff
@@ -169,23 +172,32 @@ TEST_F(TransformationTestsF, ConvertMaxPool8ToMaxPool1) {
 }
 
 TEST_F(TransformationTestsF, ConvertMaxPool14ToMaxPool8_ceil_torch_to_ceil) {
-    model = create_v14_model(ov::op::RoundingType::CEIL_TORCH, ov::Shape{1, 3, 64, 64});
+    model = create_v14_model(ov::op::RoundingType::CEIL_TORCH, ov::PartialShape{1, 3, 64, 64});
     model_ref = create_ceil_torch_workaround_model(ov::op::RoundingType::CEIL);
     manager.register_pass<ov::pass::ConvertMaxPool14ToMaxPool8>();
     comparator.disable(FunctionsComparator::CmpValues::ACCURACY);
     comparator.enable(FunctionsComparator::CmpValues::ATTRIBUTES);
 }
 
+TEST_F(TransformationTestsF, ConvertMaxPool14ToMaxPool8_ceil_torch_to_ceil_dynamic) {
+    model = create_v14_model(ov::op::RoundingType::CEIL_TORCH,
+                             ov::PartialShape{1, 3, ov::Dimension::dynamic(), ov::Dimension::dynamic()});
```

Comment:
Just make this test a little bit more robust, can the batch size dim be dynamic as well?
```suggestion
                             ov::PartialShape{ov::Dimension::dynamic(), 3, ov::Dimension::dynamic(), ov::Dimension::dynamic()});
```

Reply:
Done.